notebook/services/contents/filemanager.py

```diff
@@ -245,6 +245,14 @@ def _base_model(self, path):
         """Build the common base of a contents model"""
         os_path = self._get_os_path(path)
         info = os.lstat(os_path)
+        
+        try:
+            # size of file 
+            size = info.st_size
+        except (ValueError, OSError):
```

Comment:
I don't think there can be an error here - if `lstat()` returns, then we should have a valid stat object from which we can get `st_size`. But it shouldn't be a problem to be defensive in case I'm wrong.